.github/workflows/ios-build.yml

```diff
@@ -0,0 +1,34 @@
+name: IOS Build
+
+on:
+  workflow_call:
+    inputs:
+      NODE_VERSION:
+        default: "16.x"
+        type: string
+
+jobs:
+  build_ios:
+    name: Building the IPA
+    runs-on: macos-13
+
+    steps:
+      - uses: actions/checkout@v3.1.0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ inputs.NODE_VERSION }}
+
+      - name: Install npm dependencies
+        run: |
+          npm install
+
+      - name: Install Pod
+        run: |
+          cd ios
+          pod install
+          pod install
+
+      - name: Building the IPA
+        run: |
+          cd ios
+          fastlane build_app
```

Comment:
* Upload artifact is missing which contains the ios app
* Slack action is missing

Reply:
We are using build file only to verify whether the app is successfully built. We are uploading the app as part of publish workflow, where we need app for the automation.